jib-cli/README.md

```diff
@@ -0,0 +1,56 @@
+# Jib CLI
+
+<img src="https://img.shields.io/badge/status-experimental-orange">
+
+`jib` is a command-line utility for building containers images from file system content. 
+It serves as a demonstration of [Jib Core](https://github.com/GoogleContainerTools/jib/tree/master/jib-core),
+a Java library for building containers without Docker.
+
+This CLI tool is _experimental_ and its options and structure
+are almost certain to change.
+
+## Building
+
+Use the `application` plugin's `installDist` task to create a runnable installation.
+```sh
+$ ../gradlew installDist
```

Comment:
We can address this later, but because this requires installing JRE and Gradle, I think we should provide pre-compiled binaries, especially when considering Jib CLI is very general and isn't bound to Java developers.

Reply:
The distribution doesn't require Gradle, but it does require a JRE (as does Gradle and Maven).  But otherwise 100% agreed: the real end-goal is to use Graal's native-image to produce an executable, but that's not easy to do with the most recent release of Graal (yet).

Reply:
I meant by Gradle that to obtain Jib CLI, the doc says to run `./gradlew installDist`. I wonder if we can "pre-compile" `gradlew installDist` on Linux, Mac, and Windows, and distribute them (whatever it is)?

Reply:
Ah so we talked about. We can't offer distributed binaries unless we go through an official launch process -- so that will as you mention have to be addressed later.

Reply:
👍